.vscode/mcp.json

```diff
@@ -1,58 +0,0 @@
-{
-    "mcpServers": {
-        "Home Assistant": {
-            "command": "mcp-proxy",
-            "env": {
-                "SSE_URL": "http://192.168.1.47:8123/mcp_server/sse",
-                "API_ACCESS_TOKEN": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJjYzM3YTJmZmVjOWM0MTVlYWM1OTIxZTE0ZDk5OWY5NiIsImlhdCI6MTc0NDIyODQzOSwiZXhwIjoyMDU5NTg4NDM5fQ.hNHo3gUiRHeK8VwCLKXXendqAhR89xeTex49fu8R4cQ"
```

Comment:
The inclusion of these tokens, while maybe harmless, may actually be a security issue (if your HA install is public)...

And may result in false positive security reports.

I'd suggest making a copy of this file locally (as git will try and delete it) and rotate your token